tools/code-tools/jcov.sh

```diff
@@ -72,9 +72,19 @@ function getAsmDeps() {
     local tool_prop="`echo $tool|sed "s/-/./g"`.jar"
     local tool_versioned="$tool-$asm_version.jar"
     local tool_url="$main_url/$tool/$asm_version/$tool_versioned"
+    local tool_checksum_url="$tool_url.md5"
     if [ "$asm_manual" == "true" ] ; then
       if [ ! -e $tool_versioned ] ; then
+        wget $tool_checksum_url
         wget $tool_url
+        check_md5sum=`md5sum $tool_versioned | cut -d" " -f1`
+        check_file=`cat $tool_versioned.md5`
```

Comment:
Can this be done with `md5sum -c` on the `md5` file for simplicity?

Reply:
Sadly it isnt, the md5 from the maven repo, only contains the checksum, and not the checksum followed by the filename, required by md5sum -c ... le sigh.

Reply:
Unfortunate - ok I'll approve for now